pennylane/ops/qubit/parametrized_hamiltonian.py

```diff
@@ -0,0 +1,198 @@
+"""
+This submodule contains the time-dependent hamiltonian class
+"""
+import pennylane as qml
+from pennylane.operation import Observable, Tensor
+from pennylane.ops.qubit.hamiltonian import Hamiltonian
+
+
+# pylint: disable= too-many-instance-attributes
+class ParametrizedHamiltonian(Observable):
+    r"""Operator representing a time-dependent Hamiltonian.
```

Comment:
```suggestion
    r"""Operator representing a parametrized (e.g., time-dependent) Hamiltonian.
```
Since we don't necessarily have to have time dependence.